### ISSUE TITLE
No accessible name - Navigation Menu Links/Text

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#71]
This element has no HTML markup that would provide an accessible name for the element. A screen reader would either read nothing for this element or may make a guess.
